ios/FluentUI/Tooltip/TooltipView.swift

```diff
@@ -368,24 +394,25 @@ class TooltipView: UIView {
         return inset
     }
 
+    private weak var anchorView: UIView?
+    private weak var tokenSet: TooltipTokenSet?
```

Comment:
Thinking out loud: it's probably important to keep `anchorView` as a `weak` pointer, but `tokenSet` might be fine staying strong? The ownership model is that views own `tokenSet`s, but there should never ever be strong references in the reverse direction, so it's presumably safe to keep strong and not have to unwrap? But I don't personally have a strong opinion one way or the other.